package.json

```diff
@@ -112,6 +112,7 @@
 	"stylelint": {
 		"extends": "stylelint-config-xo",
 		"rules": {
+			"selector-type-no-unknown": null,
```

Comment:
@sindresorhus what do you say about this rule? Should it be configured to ignore customElements or should each repo have its own list of known types like it happens for JavaScript "`globals`"?

```
 51:46  ✖  Unexpected unknown type selector "g-emoji"  selector-type-no-unknown
```